src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -48,6 +51,14 @@ public boolean isSameTag(Tag otherTag) {
                 && otherTag.getTagName().equals(getTagName());
     }
 
+    public String getRemark() {
+        return remark.toString();
+    }
```

Comment:
I think it is better to make `getRemark` simply return a `Remark` instead of doing `toString()` here.